src/test/java/com/dropbox/core/DbxPKCEWebAuthTest.java

```diff
@@ -147,6 +147,22 @@ public void testScope() throws Exception {
         assertEquals(params.get("scope"), Collections.singletonList("account.info.read"));
     }
 
+    @Test
+    public void testIncrementalOAuth() throws Exception {
+        DbxPKCEWebAuth auth = new DbxPKCEWebAuth(CONFIG, APP);
+
+        String authUrl = auth.authorize(
+            DbxWebAuth.newRequestBuilder()
+                .withNoRedirect()
+                .withScope("account.info.read")
+                .withIncludeGrantedScopes(IncludeGrantedScopes.USER)
```

Comment:
Looks like this is the first attempt, this should fail if you include on the first attempt